docs/pages/reference/js/getFrameMessage.mdx

```diff
@@ -20,7 +20,16 @@ console.log(frameMessage);
   requesterFollowsCaster: false,
   likedCast: false,
   recastedCast: false,
-  requesterVerifiedAddresses: [ '0x8d25687829d6b85d9e0020b8c89e3ca24de20a89' ],
+  requesterEthAddresses: [
```

Comment:
can we just make an additional field for custody addresses and have backwards compat?

Reply:
good idea